geode-book/master_middleman/source/subnavs/geode-subnav.erb

```diff
@@ -2206,7 +2206,7 @@ gfsh</a>
                         </ul>
                     </li>
                     <li class="has_submenu">
-                        <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_apis_compatible_with_redis.html"><%=vars.product_name%> APIs compatible with Redis</a>
+                        <a href="/docs/guide/<%=vars.product_version_nodot%>/tools_modules/geode_apis_compatible_with_redis.html"><%=vars.product_name%> APIs compatible with Redis (Experimental)</a>
```

Comment:
One nit I shoulda picked in the last review: Please capitalize "Compatible" to match the page title.
Otherwise fine. Thanks!

Reply:
will change.  thx for noticing